backend/app/Models/User.php

```diff
@@ -50,8 +50,6 @@ class User extends Authenticatable
      */
     public static function createRules() {
```

Comment:
We should remove the createRules and updateRules methods, since they don't really do anything

Reply:
REmoved.